ui/pages/settings/security-tab/profile-sync-toggle/profile-sync-toggle.tsx

```diff
@@ -31,14 +30,14 @@ import { getUseExternalServices } from '../../../../selectors';
 
 function ProfileSyncBasicFunctionalitySetting() {
   const basicFunctionality: boolean = useSelector(getUseExternalServices);
-  const { setIsProfileSyncingEnabled } = useSetIsProfileSyncingEnabled();
+  const { disableProfileSyncing } = useDisableProfileSyncing();
```

Comment:
Nice, this hook will disable profile syncing and disable notifications.

FUTURE - we probs want to clean up the hooks.